### ISSUE TITLE
URL editing popout is hidden in rich text edit panel

### ISSUE THREAD

**Issue by syyong** [wagtail/wagtail#8493]
### Issue Summary

UI bug in the Wagtail admin. On our orderable edit panel, when a user is editing a URL in a richtext on the final line, the URL editing popout is almost impossible to interact with. The popout is "below" the boundary of the block/orderable bounding element:
![Screenshot 2022-05-06 at 10 25 16](https://user-images.githubusercontent.com/12908907/167105107-50fb45bb-693d-4819-9d33-5f9382936d91.png)

### Technical details

* Python version: 3.9
* Django version: 3.2
* Wagtail version: 3.0rc3
* Browser version: Firefox 99.01

**Comment by gasman** [comment]
Thanks for the report! I've been able to replicate the issue on [bakerydemo](https://github.com/wagtail/bakerydemo/) by editing the BlogPeopleRelationship class in bakerydemo/blog/models.py as follows:

```python
from wagtail.core.fields import RichTextField, StreamField

class BlogPeopleRelationship(Orderable, models.Model):
    """
    This defines the relationship between the `People` within the `base`
    app and the BlogPage below. This allows People to be added to a BlogPage.

    We have created a two way relationship between BlogPage and People using
    the ParentalKey and ForeignKey
    """
    page = ParentalKey(
        'BlogPage', related_name='blog_person_relationship', on_delete=models.CASCADE
    )
    people = models.ForeignKey(
        'base.People', related_name='person_blog_relationship', on_delete=models.CASCADE
    )
    bio = RichTextField(blank=True)
    panels = [
        SnippetChooserPanel('people'),
        FieldPanel('bio'),
    ]
```
![Screenshot 2022-05-06 at 10 33 39](https://user-images.githubusercontent.com/85097/167106462-f078c8fa-0486-4f1d-b948-bf2fcd5559f1.png)

then running `manage.py makemigrations` / `manage.py migrate`.

The same issue is present in 2.16.2, so this isn't a 3.0 regression.

**Comment by thibaudcolas** [comment]
As far as I can see this has been fixed in Wagtail 4.0 (available as `wagtail==4.0rc1` on PyPI), so will close this now. This was most likely an issue with InlinePanel hidden overflow, removed in #8983.
